content/blog/2024/bofs-and-poster-presentations.md

```diff
@@ -0,0 +1,47 @@
+Title: Share, Learn, and Connect: Introducing BoF Sessions and Poster Presentations at PyCon India 2024
```

Comment:
```suggestion
Title: Share, Learn, and Connect: Introducing BoFs Sessions and Poster Presentations at PyCon India 2024
```

Reply:
Why "BoFs" ?

Reply:
Ah! I see it's followed by Sessions. Skip this